ldap-authenticator/src/main/java/org/xwiki/contrib/ldap/XWikiLDAPConfig.java

```diff
@@ -325,6 +331,11 @@ public String getLDAPParam(String name, String cfgName, String def)
             param = this.cfgConfigurationSource.getProperty(cfgName);
         }
 
+        // Look in the default configuration that might be provided by the enabled authenticator
+        if (param == null) {
```

Comment:
> I want a fallback to the def only if my authenticator is not providing any default in the map.

This does not match the API documentation which indicate that if the searched property is not configured then you will get the passed default value. What you describe and you introduced does not behave like defaults at all, it's a final level of configuration so you should rename it and document it as such.